src/components/nav-bar/NavBar.module.scss

```diff
@@ -0,0 +1,52 @@
+@use '../../styles/config/variables' as *;
+
+.navBar {
+	position: fixed;
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+	padding: var(--spacing-xs) var(--spacing-xxl);
+	background-color: var(--color-neutral-4);
+	bottom: 0;
+	left: 0;
+	right: 0;
+
+	.addButton {
+		box-sizing: content-box;
+		background: $gradient-accent;
+		height: var(--spacing-xxl);
+		width: var(--spacing-xxl);
+		padding: var(--spacing-sm);
+		border-radius: var(--border-radius-inner);
+		color: var(--color-neutral-4);
+	}
+
+	a {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		flex-direction: column;
```

Comment:
Usually I make a mixin for flex with this as a default, then have parameters as optional overrides when needed. You can do it if you want